external/storm-solr/src/main/java/org/apache/storm/solr/bolt/SolrUpdateBolt.java

```diff
@@ -92,11 +94,19 @@ private void ack(Tuple tuple) throws SolrServerException, IOException {
         if (commitStgy == null) {
             collector.ack(tuple);
         } else {
-            toCommitTuples.add(tuple);
-            commitStgy.update();
-            if (commitStgy.commit()) {
+            if (TupleUtils.isTick(tuple)) {
+                LOG.debug("TICK! forcing solr client commit");
+                collector.ack(tuple);
+                commitStgy.commit();
```

Comment:
Shouldn't we make it consistent with the else statement?
I mean, shouldn't we check the return value of commitStgy.commit() and do other actions?